### ISSUE TITLE
Dupe with chisel 2...

### ISSUE THREAD

**Issue by ShadowT1me** [Chisel-2/Chisel-2#367]
video - http://youtu.be/0G-tB25kObY

chisel version - 2.4.1
forge - 1448

**Comment by Drullkus** [comment]
Video is private -.-

**Comment by ShadowT1me** [comment]
wait, it`s just uploading ~8 min more.

**Comment by ShadowT1me** [comment]
uploaded - https://youtu.be/0G-tB25kObY

**Comment by Drullkus** [comment]
Ok, this looks like a problem with lag and desync. I sent the video to @TheCricket and @tterrag1098 and redacted the links from this ticket.

**Comment by TheCricket** [comment]
I don't think this is our fault

**Comment by ShadowT1me** [comment]
But we think it is your troble, because problem is with interact. This dupe work with any block/mechanism where is possible put chisel

**Comment by tterrag1098** [comment]
@ShadowT1me To me, this looks like a clientside glitch. Are those diamond blocks in the end actually placeable? You never showed in your video.

**Comment by ShadowT1me** [comment]
yes. If you want i can make video v2 where i show that it is not phantom items, phantom only chisel ( like it is in my inventory but in true not )

Look our debug that we made by plugin:
[01:32:46 INFO]: Player: CraftPlayer{name=Shadow}
[01:32:46 INFO]: Item: ItemStack{AIR x 0}
[01:32:46 INFO]: Inv open: false
[01:32:46 INFO]: Inv name: net.minecraft.inventory.ContainerPlayer
[01:32:46 INFO]: =======debug=======
[01:32:47 INFO]: =======debug=======
[01:32:47 INFO]: Player: CraftPlayer{name=Shadow}
[01:32:47 INFO]: Item: ItemStack{CHISEL_CHISEL x 1, UNSPECIFIC_META:{meta-type=UNSPECIFIC}}
[01:32:47 INFO]: Inv open: false
[01:32:47 INFO]: Inv name: net.minecraft.inventory.ContainerPlayer
[01:32:47 INFO]: =======debug=======
[01:32:47 INFO]: =======debug=======
[01:32:47 INFO]: Player: CraftPlayer{name=Shadow}
[01:32:47 INFO]: Item: ItemStack{AIR x 0}
[01:32:47 INFO]: Inv open: true
[01:32:47 INFO]: Inv name: com.cricketcraft.chisel.inventory.ContainerChisel

**Comment by CptSpaceToaster** [comment]
We don't doubt that you duplicated an item.  The issue is that the method by which you duplicated the item makes it look like a really weird race condition where one client sends packets, and another doesn't receive them at the right time/properly. 

These sorts of issues can depend on a a large number of things, including issues in your network setup, maximum data-transfer speed, latency, odd issue in minecraft/forge, OR an issue in chisel.

If it's an issue in chisel... you'd need to show that you can repeat the bug/dupe on a number of different PC's, preferably with a local server also running on the same PC to avoid network issues.

**Comment by ShadowT1me** [comment]
no problem, you can check it on your PC :) or i can show it on your server - no problem. This work in any PC only thing that need like in debug in the same time put and open it ( right mouse click x2 ).

I warn you that this dupe work 100% and if you dont want to fix, just say that, but prevent people who download your mod that in your mod is hot and dangerous dupe and install mod only with own risk.

**Comment by Radviger** [comment]
Bump! I tried this and notifyed that this dupe works well! We'll waiting a fix for this...
